docs/data/trigger-sync.md

```diff
@@ -0,0 +1,270 @@
+    "sync_interval_mins": 0.1,
+    "tags": []
+  },
+  "name": "Data-Management-Service",
+  "type": "data_manager"
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+{{% expand "Click to view a full configuration example" %}}
+
+```json
+{
+  "components": [
+    {
+      "type": "camera",
+      "namespace": "rdk",
+      "attributes": {
+        "video_path": "FDF90FEB-59E5-4FCF-AABD-DA03C4E19BFB"
+      },
+      "depends_on": [],
+      "service_configs": [
+        {
+          "type": "data_manager",
+          "attributes": {
+            "capture_methods": [
+              {
+                "capture_frequency_hz": 0.2,
+                "method": "ReadImage",
+                "additional_params": {
+                  "mime_type": "image/jpeg"
+                }
+              }
+            ]
+          }
+        }
+      ],
+      "name": "webcam",
+      "model": "webcam"
+    },
+    {
+      "attributes": {
+        "start": "14:10:00",
+        "end": "15:35:00",
+        "zone": "CET"
+      },
+      "depends_on": [],
+      "name": "timesensor",
+      "model": "naomi:sync-at-time:timesyncsensor",
+      "type": "sensor",
+      "namespace": "rdk"
+    }
+  ],
+  "services": [
+    {
+      "attributes": {
+        "selective_syncer_name": "timesensor",
+        "sync_interval_mins": 0.2,
+        "capture_dir": "",
+        "tags": [],
+        "additional_sync_paths": []
+      },
+      "name": "datamanager",
+      "type": "data_manager",
+      "namespace": "rdk"
+    }
+  ],
+  "agent_config": {
+    "subsystems": {
+      "viam-server": {
+        "release_channel": "stable",
+        "pin_version": "",
+        "pin_url": "",
+        "disable_subsystem": false
+      },
+      "agent-provisioning": {
+        "release_channel": "stable",
+        "pin_version": "",
+        "pin_url": "",
+        "disable_subsystem": false
+      },
+      "viam-agent": {
+        "release_channel": "stable",
+        "pin_version": "",
+        "pin_url": "",
+        "disable_subsystem": false
+      }
+    }
+  },
+  "modules": [
+    {
+      "module_id": "naomi:sync-at-time",
+      "version": "0.0.2",
+      "type": "registry",
+      "name": "naomi_sync-at-time"
+    }
+  ]
+}
+```
+
+{{% /expand%}}
+
+You have now configured sync to happen during a specific time slot.
+
+## Test your sync configuration
+
+To test your setup, [configure a webcam](https://docs.viam.com/components/camera/webcam/) or another component and [enable data capture on one of the component methods](https://docs.viam.com/data/capture/#configure-data-capture-for-individual-components), for example `ReadImage`.
+The data manager will now capture data.
+Go to the [**Control** tab](https://docs.viam.com/fleet/machines/#control).
+You should see the sensor.
+Click on `GetReadings`.
+
+{{<imgproc src="/data/timesensor.png" resize="800x" declaredimensions=true alt="Control tab with sensor panel">}}
+
+If you are in the time frame for sync, the sensor will return true.
```

Comment:
```suggestion
If you are in the time frame for sync, the time sync sensor will return true.
```
[opt] Maybe overkill but in case someone is syncing sensor data, to disambiguate